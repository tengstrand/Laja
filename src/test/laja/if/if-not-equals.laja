#write "target/test-output/if-equals.out"
  #set (x = 1)
  #if (x != 2)
x is not 2
  #end
#end