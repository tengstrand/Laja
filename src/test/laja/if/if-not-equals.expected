x is not 2
